atom/browser/importer/profile_writer.cc

```diff
@@ -26,7 +28,13 @@
 
 namespace importer {
 void ShowImportLockDialog(gfx::NativeWindow parent,
-                          const base::Callback<void(bool)>& callback) {}
+                          const base::Callback<void(bool)>& callback) {
```

Comment:
don't we have to call this callback with `is_continue` for `OnImportLockDialogEnd`? Seems like we should send the callback in the emit